core/new-gui/src/app/workspace/service/workflow-graph/model/shared-model-change-handler.ts

```diff
@@ -262,19 +262,34 @@ export class SharedModelChangeHandler {
             // Changes one level below the operatorPredicate type
             for (const entry of event.changes.keys.entries()) {
               const contentKey = entry[0];
-              if (contentKey === "isCached") {
+              if (contentKey === "viewResult") {
                 const newCachedStatus = this.texeraGraph.sharedModel.operatorIDMap
                   .get(operatorID)
-                  ?.get("isCached") as boolean;
+                  ?.get("viewResult") as boolean;
                 if (newCachedStatus) {
-                  this.texeraGraph.cachedOperatorChangedSubject.next({
-                    newCached: [operatorID],
-                    newUnCached: [],
+                  this.texeraGraph.viewResultOperatorChangedSubject.next({
+                    newViewResultOps: [operatorID],
+                    newUnviewResultOps: [],
```

Comment:
as the change subject can accept a list of `newViewResultOps` and a list of `newUnviewResultOps`, shall we update them as a group rather than one by one in a for loop?

Reply:
it seems that this code is already doing it at a list level, so it's resolved